test/integration/integration_test.go

```diff
@@ -402,19 +407,15 @@ func ensureTestDirEmpty() {
 	}
 
 	for _, file := range files {
-		fullPath := filepath.Join(testDir, file.Name())
-		if filepath.Base(file.Name()) == "cache" {
-			fmt.Println(fmt.Sprintf("Keeping Minishift cache directory '%s' for test run.", fullPath))
-			continue
-		}
-		os.RemoveAll(fullPath)
+		os.RemoveAll(filepath.Join(testDir, file.Name()))
 	}
 }
 
 func cleanTestDirConfiguration() {
 	var foldersToClean []string
 	foldersToClean = append(foldersToClean, filepath.Join(testDir, "addons"))
 	foldersToClean = append(foldersToClean, filepath.Join(testDir, "config"))
+	foldersToClean = append(foldersToClean, filepath.Join(testDir, "cache"))
```

Comment:
This will slow the runs a little bit because we will download the `oc` on each time, but nothing critical I think. Alternative could be to delete just `cache/images`.